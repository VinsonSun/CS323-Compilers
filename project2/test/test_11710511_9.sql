int max(int a, int b) {
   if(a > b) {
      return a;
   } else {
      return b;
   }
}

int test9() {
   int a = 1;
   int b = 2;
   int c = 3;
   return max(a, b, c);
}
