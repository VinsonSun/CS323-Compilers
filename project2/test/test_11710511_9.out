Error type 9 at Line 13: invalid argument number for max, expect 2, got 3
